### ISSUE TITLE
WCAG 3.1.1: Ensures every HTML document has a lang attribute (html)

### ISSUE THREAD

**Issue by VitthalBobade** [VitthalBobade/accessibilityinsightsreports#2]
#### Issue
`Ensures every HTML document has a lang attribute` ([html-has-lang](https://dequeuniversity.com/rules/axe/3.5/html-has-lang?application=msftAI))
#### Target application
[BuddyConnect Admin Console | Login](https://buddyconnectwebdev.azurewebsites.net/)
#### Element path
html
#### Snippet
`<html>`
#### How to fix
    Fix any of the following:
      The <html> element does not have a lang attribute
#### Environment
Chrome version 84.0.4147.105
This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
